lib/oauth2/response.ex

```diff
@@ -46,6 +46,7 @@ defmodule OAuth2.Response do
     Enum.map(headers, fn {k, v} -> {String.downcase(k), v} end)
   end
 
+  defp decode_response_body(nil, _type, _), do: nil
```

Comment:
```suggestion
  defp decode_response_body(nil, _type, _), do: ""
```

I am wondering if we should return an empty string instead here. Any thoughts?

Reply:
I don't have a strong feeling. In practice I'm not sure it would matter. Happy to update if you think it's preferred.

Reply:
> Plug doesn't seem to support building a response with a nil body

You mentioned that. Should we return an empty string? It seems that they use `nil` whenever the response is sent.

I am opted in for empty string

Reply:
@yordis I updated this to return an empty string.